packages/lime-docs/files/lime-example

```diff
@@ -51,6 +51,7 @@ config lime network
 	option bmx7_over_batman false
 	option bmx7_pref_gw none			# Force bmx7 to use a specific gateway to Internet (hostname must be used as identifier)
 	option bmx7_wifi_rate_max 'auto'
+	option babeld_over_batman false			# When Babeld is run without VLAN (babeld:0), it runs on the bridge which includes Batman-adv's bat0, keeping this false avoids to have Babeld metrics distorted by Batman-adv
```

Comment:
@ilario When someone would want to change from `false` to `true` ? If there is not known useful use case I prefer not to add more options.

Reply:
I agree with you, and I'll remove this.
I added this originally just because the same was present for BMX6 over Batman-adv